ui/app/AppLayouts/Chat/ChatColumn/Message.qml

```diff
@@ -130,7 +130,7 @@ Item {
         Item {
             visible: chatsModel.activeChannel.chatType !== Constants.chatTypePrivateGroupChat || chatsModel.activeChannel.isMember
             id: wrapper
-            height: wrapper.visible ? fetchMoreButton.height + fetchDate.height + 3 + Style.current.smallPadding*2 : 0
+            height: wrapper.visible ? childrenRect.height + Style.current.smallPadding*2 : 0
```

Comment:
Did you check to see if that didn't create a binding loop?

Reply:
Yup. No binding loops!